### ISSUE TITLE
Een grondboring is niet 

### ISSUE THREAD

**Issue by gezever** [Informatievlaanderen/OSLOthema-bodemEnOndergrond#121]
**Grondboringen* 

**Omschrijving van het probleem:** 
In het applicatieprofiel is een boorgat het resultaat van een grondboring, subklasse van een sosa bemonstering.

**Omschrijving van een mogelijke oplossing:**
Het resultaat van het boren van een gat is dat de toestand van het gat veranderd, breder/dieper wordt. Een grondboring is dus in eerste plaats een sosa:Actuation, de boor is daarbij een sosa:Actuator
Een boorgat is een  [sosa:FeatureOfInterest](https://www.w3.org/TR/vocab-ssn/#SOSAFeatureOfInterest) van een sosa:Actuation.

zie https://github.com/bodemenondergrond/grondverzet

**Comment by GeertThijs** [comment]
- Kan wel zijn maar een Sampling heeft een Sample als resultaat (Sample_Feature bij ISO, verder opgesplitst in SpatialSamplingFeature en Specimen,), waarvan Borehole (via de subklasse SpatialSamplingFeature) een subklasse is. Dus leek het perfect logisch om Grondboring te beschouwen als een Sampling. Vooral omdat Actuations helemaal geen Samples als resultaat kunnen hebben (zie tweede punt).
- Actuations veranderen de waarde van een kenmerk van een bestaand object, bvb een waterkraan gaat van open naar gesloten. Resultaat is een andere waarde, geen nieuw object zoals bij Sampling waar bvb een Specimen ontstaat of een bemonsteringsplaats is aangeduid  (zijnde bvb een Borehole of Observation-Well of Station of Sounding of Outcrop etc etc). En zeker voor Grondboringen willen we dat de Boring een Boorgat oplevert ipv dat het de grond ergens dieper maakt.

**Comment by gezever** [comment]
@GeertThijs 
Je vergist je wanneer je beweert dat wij willen dat een boring een boorgat oplevert. Ik zeg net dat we dat niet willen.

**Comment by GeertThijs** [comment]
> @GeertThijs Je vergist je wanneer je beweert dat wij willen dat een boring een boorgat oplevert. Ik zeg net dat we dat niet willen.

We willen toch kunnen zeggen welke activiteit verantwoordelijk is voor het Boorgat? Actuations kunnen dat niet, dat is een Activiteit die een kenmerk van een bestaand object verandert. In dit geval zou dat het kenmerk "aangeboord" of zoiets van een object Perceel zijn, waarbij de waarde van false op true komt. Samplings kunnen dat wel, resultaat is nl een Sample (of SamplingFeature bij ISO), waarvan Borehole een subklasse is. Hier Actuation toepassen zou een verkeerd gebruik van die klasse zijn.

**Comment by gezever** [comment]
@GeertThijs 
Het boren van een waterput is geen sampling. 
Meer info: https://github.com/bodemenondergrond/grondverzet/blob/main/README.md

**Comment by GeertThijs** [comment]
Klopt, maar de oplossing (Actuation) is erger dan de kwaal. Ik heb de usageNote wat aangepast. In de werkgroepen is wel altijd duidelijk geweest dat het om Boiringen met het oog op het nemen van Monsters of het uitvoeren van Boorgatmetingen ging.

**Comment by gezever** [comment]
Dit heeft geen zin meer.
